### DISCUSSION TITLE
Can i connect budibase to mysql on xampp?

### DISCUSSION THREAD

**Discussion by Asonyu** [Budibase/budibase#2768] [Help]
Hello, please i need help am running xampp server and am trying to connect my mysql database to budibase but i keep getting this error
![Uploading Screen Shot 2021-09-28 at 11.57.02.png…]()

**Comment by mjashanks** [comment]
Hey @Asonyu - your image has not been uploaded, could you try again please.

**Comment by Pachat** [comment]
I don't see your image but I can answer YES. Here are my settings
```
Host: host.docker.internal
Port: 3306
User: root
Password: *****
Database: bbtest
```
